apps/frontend/src/renderer/components/settings/integrations/IntegrationCard.tsx

```
@@ -0,0 +1,432 @@
+                                        <span className="truncate">{integration.description}</span>
+                                    </>
+                                )}
+                            </div>
+                        </div>
+
+                        {/* Expand button */}
+                        <button
+                            onClick={() => setIsExpanded(!isExpanded)}
+                            className="p-1 hover:bg-muted rounded"
+                        >
+                            {isExpanded ? (
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)